modules/generic-extras/src/main/scala/io/circe/generic/extras/ConfiguredJsonCodec.scala

```diff
@@ -15,6 +15,7 @@ private[generic] class ConfiguredJsonCodecMacros(val c: blackbox.Context) extend
   import c.universe._
 
   protected[this] def semiautoObj: Symbol = symbolOf[semiauto.type].asClass.module
+  protected[this] def deriveFunctionPrefix: String = "deriveConfigured"
```

Comment:
This is super nitpicky but I'd probably call this `deriveMethodPrefix`, since it's a method, not a function. I'm happy either to leave it or to change it in a follow-up, though.

Reply:
I considered "derive*" functions, as they are not bound to a class instance (in the OOP sense). I'll change it real quick 🙂